packages/sdk/test/unit/apiClient/apiClient.spec.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { assert } from "chai";
+import { MockAuthProvider } from "./mockAuthProvider";
+import axios from "axios";
+import { createApiClient } from "../../../src";
+import MockAdapter from "axios-mock-adapter";
+
+describe("ApiClient Tests - Node", () => {
+  it("createApiClient should return axios instance with base url and config updated", async function () {
+    // Arrange
+    const apiBaseUrl = "https://fake-api-endpoint";
+    const mockAuthProvider = new MockAuthProvider();
+    const axiosMockAdapter = new MockAdapter(axios);
```

Comment:
not sure whether this is a special case, but seems changes to axios also impacts the new axios instance we created. From feature perspective, we may expect the new instance is a brand new instance that do not have any customization that made by developers somewhere else.

We don't need to resolve this issue in this PR. Let's add a task to double confirm the behavior and see whether we need some changes.